db/schema.rb

```diff
@@ -43,7 +43,7 @@
     t.string "offer_withdrawal_reason"
     t.datetime "offer_withdrawn_at"
     t.datetime "sent_to_provider_at"
-    t.jsonb "structured_rejection_reasons"
+    t.text "structured_rejection_reasons"
```

Comment:
Why is this a thing?

Reply:
We were thinking about serialising to YAML instead of JSON

Reply:
How come it's coming is as a migration on my PR? 😳

Reply:
These get updated on `master` when you run `db:migrate`, I won't make this changest part of this PR!

Reply:
Must have been a rebase snafu after https://github.com/DFE-Digital/apply-for-teacher-training/pull/2336/files. I'll raise a PR